m2m-transfers/V3/REPWRITERSPECS/BANNO.M2MTRANSFERS.V3.POW

```diff
@@ -1171,6 +1177,9 @@ PROCEDURE GETRECIPSLDATA
        NEWTRANSLDESCRIPTION=SHARE:NICKNAME
        IF NEWTRANSLDESCRIPTION="" THEN
         NEWTRANSLDESCRIPTION=SHARE:DESCRIPTION
+       CHARCHECK=NEWTRANSLDESCRIPTION
```

Comment:
Instead of doing the checking/scrubbing here, can we just do the check where we are outputting the json?

Reply:
@rrobison44 : removed